.prod.env

```diff
@@ -110,7 +106,11 @@ MAIL_USERNAME=fixme
 #MAX_DOWNLOAD_ARCHIVE_SIZE=1024 * 1024 * 1024  # max total files size for archive download
 
 #USE_X_ACCEL=False  # use nginx (in front of gunicorn) to serve files (https://www.nginx.com/resources/wiki/start/topics/examples/x-accel/)
+USE_X_ACCEL=1
 
+# where geodiff lib copies working files
+#GEODIFF_WORKING_DIR=$LOCAL_PROJECTS/geodiff_tmp
+GEODIFF_WORKING_DIR=/data/geodiff
```

Comment:
do we need this folder?

Reply:
I thought it would be better if it is explicitly stated.